server/utils.js

```diff
@@ -0,0 +1,24 @@
+// Compute the distance on surface of earth given pair
+// of (lat, long) geo-locations
+exports.geoDist = (lat1, lon1, lat2, lon2) => {
+  // script taken from: https://www.movable-type.co.uk/scripts/latlong.html
+  const R = 6371e3; // metres
+  let φ1 = (lat1 * Math.PI) / 180; // φ, λ in radians
```

Comment:
I do not quite suggest using special character as variable name even tough this looks pretty cool. 
I am not sure whether this special character can be correctly shown in all editor.